core-it-support/core-it-bootstrap/pom.xml

```diff
@@ -19,45 +19,58 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I think from a history perspective, this shouldn't be moved, but all groups removed a new module added.

Reply:
Could you rephrase please, I don't really understand...

Reply:
Sure, when looking at the history it seems to me that the group 13 have been rewritten to a plugin. Logically that is not the case. All groups have been *removed* and *replaced* with a new plugin. I don't know whether Git autodetects the move or this has been done deliberately.

Better now?